client/components/App.jsx

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import axios from 'axios';
 import ReviewList from './ReviewList.jsx';
+import LeftDiv from './LeftDiv.jsx';
```

Comment:
Might make more sense to have a better name than LeftDiv ie: ReviewReport or something along those lines